website/docs/developer/api/api-gateway.md

```diff
@@ -0,0 +1,105 @@
+---
+id: API gateway paths, inputs and outputs
+title: API Gateway paths, inputs and outputs
+description: Additional notes on API gateway paths, inputs and expected outputs.
+---
+
+## Overview
+
+This documentation provides details on the paths, input and outputs for the `orca_api` API gateway. The format for the API invoke URL is `https://example.execute-api.us-west-2.amazonaws.com/[stage_name]/[path]`. Check [API gateway research webpage](https://github.com/nasa/cumulus-orca/blob/master/website/docs/developer/research/research-APIGateway.md) for more information.
+
+## Path 
+- The path `catalog/reconcile` of `orca_api` API gateway returns additional metadata information needed for reporting by triggering the `orca_catalog_reporting` lambda.
```

Comment:
We do not need to relay what components or how we are doing things under the covers. We want to relay what API call is for. In this example something like `The catalog/reconcile API call provides a user with a current listing of the ORCA catalog that can be used to reconcile granule and file information against a master catalog. For example, comparing the Cumulus primary archive holdings against the ORCA holdings to find discrepancies.`